internal/api/sms_provider/twilio.go

```diff
@@ -70,7 +80,9 @@ func (t *TwilioProvider) SendSms(phone, message, channel string) (string, error)
 	receiver := "+" + phone
 	if channel == WhatsappProvider {
 		receiver = channel + ":" + receiver
-		sender = channel + ":" + sender
+		if !isTwilioMessagingServiceID(sender) {
+			sender = channel + ":" + sender
```

Comment:
so if the message service id is using the incorrect format, then the sender here will be `whatsapp:phone_number` right? if it's using the correct format, then the sender is just the message service sid? 

is it stated in twilio's docs that the message service sid can be a phone number? the reason is because if users are setting the `MessageServiceSid` as a phone number as a workaround, then this would be a hacky fix which isn't ideal. 

i feel like just based on the code alone, it's not immediately obvious why the sender is modified if the `isTwilioMessagingServiceID` fails.

Reply:
We currently support setting  phone number as a MessageServiceSid on the `sms` channel. Consequently, we allow setting a phone number on the WhatsApp channel for consistency with that.

I can see why the current state is unclear though, as there's an implicit assumption that it's a phone number if it's not a MessagingServiceSID. Let's change the check to catch for phone numbers instead of SIDs